community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/CommunityPipeBuilder.scala

```diff
@@ -138,7 +138,16 @@ case class CommunityPipeBuilder(monitors: Monitors, recurse: LogicalPlan => Pipe
         val predicate = predicates.map(buildPredicate).reduceOption(_ andWith _).getOrElse(True())
         OptionalExpandIntoPipe(source, fromName, relName, toName, dir, LazyTypes(types), predicate)(id = id)
 
-      case VarExpand(_, IdName(fromName), dir, projectedDir, types, IdName(toName), IdName(relName), VarPatternLength(min, max), expansionMode, predicates) =>
+      case VarExpand(_,
+                     IdName(fromName),
+                     dir,
+                     projectedDir,
+                     types,
+                     IdName(toName),
+                     IdName(relName),
+                     VarPatternLength(min, max),
+                     expansionMode,
+                     _, _, _, _, predicates) =>
```

Comment:
This could be where to start, if the new predicates should be used in community instead of legacy predicates.